README.md

```diff
@@ -98,6 +98,25 @@ All directive name should be prepended with `v-`, `v-morph-url` like so.
 
 # Morphling Filters
 
+### morph-age
+
+> `import { morphAge } from 'vue-morphling';`
+
+```html
+<p>{{ new Date(1984, 8, 25) | morph-age }}</p>
+<!-- Numb -->
+```
+
+```javascript
+const birthday = new Date(1984, 8, 25);
```

Comment:
Also this one @rixlabs if it's not too much, maybe you could use Chester Bennington's birthday for consistency of the examples? It's March 20, 1976.